src/stories/Library/Forms/checkbox/checkbox.scss

```diff
@@ -122,4 +124,8 @@
     margin: 0;
     height: auto;
   }
+
+  .dpl-input.checkbox-override .input-label {
+    cursor: pointer;
```

Comment:
@Dresse would hitting the label actually trigger the checkbox to get checked and unchecked? I intentionally left it without cursor pointer because I thought that in Drupal we can't get that functionality. So I just thought it would be confusing for the user to think they can click the label but then nothing actually happens.

Reply:
Yes, clicking the label is checking/unchecking the checkbox. The same goes for the radio buttons 😄